src/Sulu/Bundle/SecurityBundle/Resources/public/js/components/users/components/form/main.js

```diff
@@ -232,6 +233,16 @@ define(['app-config', 'widget-groups'], function(AppConfig, WidgetGroups) {
             ]);
         },
 
+        bindFormEvents: function() {
```

Comment:
@nnatter wouldn't it be a nice idea to bind a blur event and if the inputs are empty reset the placeholder